datafusion/sqllogictest/test_files/array.slt

```diff
@@ -7013,6 +7013,12 @@ select array_resize(arrow_cast([[1], [2], [3]], 'LargeList(List(Int64))'), 10, [
 ----
 [[1], [2], [3], [5], [5], [5], [5], [5], [5], [5]]
 
+# array_resize null value
+query ?
+select array_resize(arrow_cast(NULL, 'List(Int8)'), 1);
+----
+NULL
```

Comment:
shouldn't it be 
[NULL] 

@jonathanc-n

Reply:
i think it's fine. the query creates no array (besides null), so i wouldn't expect 1-element array in the output.

Reply:
` Resizes the list to contain size elements`
gotcha !

Thanks @findepi